src/Camera.py

```diff
@@ -1,15 +1,55 @@
 #!/usr/bin/python3
-
-import PySpin
-
+import logging
+logger = logging.getLogger(__name__)
+
+try:
+    import PySpin
+except ImportError:
+    PySpin = None
+    logger.warn("Could not import PySpin; using mocked cameras.")
+    
 import numpy as np
 import time, datetime
 
 
-class Camera():
+def listCameras():
+    global pyspin_cameras, pyspin_instance
+    if PySpin is not None:
+        cameras = PySpinCamera.listCameras()
+    else:
+        cameras = [MockCamera(), MockCamera()]
```

Comment:
This would also apply to the case of "pyspin installed, but no cameras connected"